### PR TITLE
Add tasks to enable redfish configuration on a Dell node

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

File: lib/jobs/redfish-enable-alerts-job.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+    // Copyright 2016, EMC, Inc
```
Comment (anhou) [inline lib/jobs/redfish-enable-alerts-job.js]: the whole file lines are shifted?

File: lib/jobs/redfish-enable-alerts-job.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            self.options = options;
+            self.context = context;
+    
+            RestJob.super_.call(self, logger, options, context, taskId);
+            self.restClient = new HttpTool();
+        }
+    
+        util.inherits(RestJob, BaseJob);
+    
+        RestJob.prototype._run1 = function run(){
```
Comment (anhou) [inline lib/jobs/redfish-enable-alerts-job.js]: run1?

**Comment by tannoa2** [comment]
@RackHD/veyron @RackHD/corecommitters

File: lib/jobs/redfish-enable-alerts-job.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
+// Copyright 2016, EMC, Inc
```
Comment (anhou) [inline lib/jobs/redfish-enable-alerts-job.js]: please correct the copyright, the same for other files

File: lib/utils/job-utils/racadm-tool.js [hunk 4]
```diff
@@ -465,5 +465,55 @@ function racadmFactory(
...
+     * Get BIOS xml file and store it to desired file path and file name
+     *
+     * @param {string} host
+     * @param {string} user
+     * @param {string} password
+     * @return {promise}
+     */
+    RacadmTool.prototype.enableAlert = function(host, user, password) {
+        return this.runCommand(host, user, password, "set iDRAC.IPMILan.AlertEnable 1")
+            .then(function(data){
```
Comment (brianparry) [inline lib/utils/job-utils/racadm-tool.js]: @tannoa2 These `.then` statements are not needed.

**Comment by iceiilin** [comment]
Is there any possibility to share the job https://github.com/RackHD/on-tasks/blob/master/lib/jobs/racadm-control.js by adding the corresponding redfish logic in https://github.com/RackHD/on-tasks/blob/master/lib/utils/job-utils/racadm-tool.js, just like https://github.com/RackHD/on-tasks/blob/master/lib/utils/job-utils/racadm-tool.js#L402?

**Comment by tannoa2** [comment]
@iceiilin it is a good suggestion. I will open a story to look into that. 
For the sake of completing this feature, I will keep as it is now since it works.

**Comment by tannoa2** [comment]
can you please, would you please, test this please

**Pull request merged**